module/VuFind/tests/unit-tests/src/VuFindTest/Marc/MarcLintTest.php

```diff
@@ -60,5 +61,90 @@ public function testMarcLint()
         $marc = $this->getFixture('marc/marclint_bad2.xml');
         $reader = new \VuFind\Marc\MarcReader($marc);
         $this->assertEquals(38, count($lint->checkRecord($reader)));
+
+        // Test checkArticle with field 130 and an invalid field:
+        $lint = new \VuFind\Marc\MarcLint();
+        $this->CallMethod(
```

Comment:
Is that capital C on CallMethod a typo? It's repeated a few times but seems odd.

Reply:
Typo, fixed.